src/main/java/com/google/devtools/build/lib/bazel/bzlmod/BazelLockFileModule.java

```diff
@@ -89,13 +89,22 @@ public void afterCommand() throws AbruptExitException {
               }
             });
 
+    BazelDepGraphValue depGraphValue;
+    try {
+      depGraphValue = (BazelDepGraphValue) evaluator.getExistingValue(BazelDepGraphValue.KEY);
+    } catch (InterruptedException e) {
+      // Not thrown in Bazel.
```

Comment:
what does this mean? Is it thrown in Blaze?

Reply:
you could work around this by calling `evaluator.getInMemoryGraph().getIfPresent(...)`.

But I think this value is now unnecessary -- you could just remove it since it's no longer used

Reply:
It is needed to learn of extensions that are no longer used anywhere. This was found by tests.

Reply:
For some reason the graph maintained by the `MemoizingEvaluator` instance I was holding onto became empty due to Skyframe. Instead, I'm now holding on the `SkyframeExecutor`, which seems to work. Looks like something in Skyfocus is switching out the evaluator after the build - I don't understand where or why this happening, but I won't complain if the tests pass.

Reply:
Nvm, found the cause now: https://github.com/bazelbuild/bazel/blob/3beaaaf23e4f6e9071ef7eabced7d64513573e80/src/main/java/com/google/devtools/build/lib/skyframe/SkyframeExecutor.java#L4077